config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -598,6 +598,25 @@ postsubmits:
         resources:
           requests:
             memory: "1Gi"
+  - name: post-test-infra-upload-triage
+    cluster: test-infra-trusted
+    branches:
+    - master
+    run_if_changed: '^triage/Makefile$|^triage/.+(\.html|\.js)$'
```

Comment:
This misses CSS files. Also `.+` can include subdirectories, it's probably better to use `[^/]`

Reply:
definitely want to catch css. shrug on subdirs, this just causes the job to run, it's cheap.